OpenRA.Mods.RA2/UtilityCommands/ImportRA2MapCommand.cs

```diff
@@ -337,437 +338,25 @@ sealed class ImportRA2MapCommand : IUtilityCommand
 			}
 		};
 
-		static readonly string[] LampActors =
-		{
-			"GALITE", "INGALITE", "NEGLAMP", "REDLAMP", "NEGRED", "GRENLAMP", "BLUELAMP", "YELWLAMP",
-			"INYELWLAMP", "PURPLAMP", "INPURPLAMP", "INORANLAMP", "INGRNLMP", "INREDLMP", "INBLULMP"
-		};
-
-		static readonly Dictionary<string, string> ReplaceActors = new()
+		// Dogs obviously don't deploy into different dogs, but there used to be a "ReplaceActors"
+		// because in OG RA2 there were two separate dog actors and we just use faction-specific sprites.
+		// Since what this does is replace actor A with actor B
+		// and add a DeployStateInit and that init doesn't hurt,
+		// we piggy-back off of it to translate to the proper actor time,
+		// even if with a redundant DeployStateInit.
+		protected override Dictionary<string, string> DeployableActors { get; } = new()
```

Comment:
Then simply keep this forked version instead inheriting from the TS artillery function. Deploying dogs is awkward.